src/Time/Time.js

```diff
@@ -156,14 +163,15 @@ class Time extends Component {
                         disabled={disabled}
                         downButtonProps={meridiemDownButtonProps}
                         inputProps={meridiemInputProps}
+                        localizedText={this.props.localizedText}
```

Comment:
In these four instances, you can just use the variable `localizedText` that you pruned from `this.props` up on line 59.